src/components/ExportPanel.vue

```diff
@@ -131,8 +132,9 @@ export default {
         timer: this.$store.state.session.timer,
         started: this.$store.state.session.started,
         ended: this.$store.state.session.ended,
+        reportLogo: this.config.reportLogo,
+        logoPath: this.config.logoPath,
```

Comment:
If we are going to have multiple fields, we should nest them:

```
reports: {
  logo: {
    enabled: true,
    path: "/home/user/logo.png"
  }
}
```

Reply:
Ok, got it.